src/com/csciOOD/Screen.java

```diff
@@ -2,98 +2,68 @@
-        contentPane.add(headerPanel, BorderLayout.NORTH);
-        contentPane.add(content, BorderLayout.CENTER);
+        setLayout(new BorderLayout());
+        setPreferredSize(new Dimension(800, 800));
 
-        setSize(800, 800);
+        // Establish base Layered Pane
+        add(basePane, BorderLayout.CENTER);
+        basePane.setBounds(0, 0, 800, 800);
 
-        // Start our gameLoop
+        // Add panes that we'll be using (name, z-index)
+        basePane.add(headerPanel, new Integer(1));
+        basePane.add(content, new Integer(0));
+        basePane.add(menu, new Integer(2));
+
+        // Start our gameLoop from our Main Game object
         Thread mainThread = new Thread(content);
         mainThread.start();
     }
 
     public static void main(String[] args) {
         Screen gameScreen = new Screen();
+        gameScreen.pack();
         gameScreen.setVisible(true);
     }
 
-    private class MainPannel extends JPanel implements Runnable {
-        float interpolation;
-        // Top-level state for control flow
-        private boolean gameRunning = false;
-        private boolean isPaused = false;
-
-        public MainPannel() {}
-
-        public void run(){
-            gameLoop();
-        }
-
-        // Any methods that change game state in here
-        public void update() {
-            gameInstance.updateGame();
-        }
-
-        // TODO: mitigate death spiral and lag scnearios
-        private void gameLoop() {
-
-            // Shoot for the Stars!
-            double fpsLimit = 120;
-            double updateInterval = 1000000000 / fpsLimit; // Should / could be set to something else. Lower? higher?
-            double renderInterval = 1000000000 / fpsLimit;
-            double lastUpdateTime = System.nanoTime();
-            double lastRenderTime = System.nanoTime();
-
-            while (gameRunning) {
-                double now = System.nanoTime();
-
-                if (!isPaused) {
-                    double deltaUpdate = now - lastUpdateTime;
-                    double deltaRender = now - lastRenderTime;
-
-                    while(deltaUpdate > updateInterval) {
-                        lastUpdateTime += updateInterval;
-                        update();
-                    }
-
-                    float interpolation = Math.min(1.0f, (float) (deltaUpdate / updateInterval));
-
-                    drawGame(interpolation);
-
-                    lastRenderTime = now;
-
-                    while (deltaRender < renderInterval && deltaUpdate < updateInterval) {
-                        Thread.yield();
-                    }
+    private void bindKeys(){
+        KeyboardFocusManager.getCurrentKeyboardFocusManager().addKeyEventDispatcher(new KeyEventDispatcher() {
+            @Override
+            public boolean dispatchKeyEvent(KeyEvent e) {
+                if (e.getID() == KeyEvent.KEY_PRESSED && e.getKeyCode() == KeyEvent.VK_ESCAPE){
+                    pauseGame();
                 }
```

Comment:
sounds good 👍    This will work just fine for all of my purposes, I just didn't want to speak for you since your use-case for this is more complicated